### DISCUSSION TITLE
Question:  Comparison of features with newly announced OOTB vs Pnp Search

### DISCUSSION THREAD

**Discussion by mikezimm** [microsoft-search/pnp-modern-search#1701] [Q&A]
The Docs page says post a question here but there is no question template so here goes:

In the [SharePoint Roadmap Pitstop: December 2021](https://techcommunity.microsoft.com/t5/microsoft-sharepoint-blog/sharepoint-roadmap-pitstop-december-2021/ba-p/3050059), they announced new customization capability.

[Manage search verticals](https://docs.microsoft.com/en-us/MicrosoftSearch/manage-verticals)

[Manage result types](https://docs.microsoft.com/en-us/MicrosoftSearch/manage-result-types)

I was just wondering if there is a good comparison showing the current differences/pros/cons between Pnp Search and the new OOTB?

**Comment by wobba** [comment]
We haven't created docs comparing the two - and it's not just a side by side comparison the way I see it. As it depends on your scenario and use case.
